src/tests/contract-tests/consumer/sis-consumer.pact.test.ts

```diff
@@ -0,0 +1,81 @@
+import path from "path";
+import { PactV4, SpecificationVersion, MatchersV3 } from "@pact-foundation/pact";
+import type { UserIdentityResponseMetadata } from "../../../handlers/user-identity/user-identity-response-metadata";
+
+const { like } = MatchersV3;
+
+const USER_IDENTITY_ENDPOINT = "/user-identity";
+
+const provider = new PactV4({
+  consumer: "SisConsumerTests",
+  provider: "StoredIdentityService",
+  logLevel: "warn",
+  dir: path.resolve(process.cwd(), "pacts"),
+  spec: SpecificationVersion.SPECIFICATION_VERSION_V4,
+  host: "127.0.0.1",
+});
+
+describe("SIS Consumer", () => {
+  it("should return 200 if the stored identity exists", async () => {
+    const govukSigninJourneyId = "Test-" + Date.now();
+
+    await provider
+      .addInteraction()
+      .given("stored identity exists")
+      .uponReceiving("A userId for a stored identity that does not exist")
+      .withRequest("POST", USER_IDENTITY_ENDPOINT, (builder) => {
+        builder.headers({
+          Authorization: like(`Bearer test-access-token`),
+        });
+        builder.jsonBody({
+          vtr: ["P2"],
+          govukSigninJourneyId,
+        });
+      })
+      .willRespondWith(200, (builder) => {
+        builder.jsonBody(
+          like({
+            content: {
+              sub: "bob-smith",
+              iss: "http://api.example.com",
+              vot: "P3",
```

Comment:
Why is `$.content.vot` (P3) higher than `$.vot` (P2) in this "success example?

My understanding of the API is that the fomrer is the calculated `vot` and the latter is the `vot` from EVCS (the maximum). So the latter should always be great than or equal to the former?

Also `$.content.vot` should always match the `vtr` on the input in the successful case, so it should be P2 I think?